documentation/src/main/asciidoc/userguide/chapters/bootstrap/Bootstrap.adoc

```diff
@@ -32,9 +30,9 @@ First is the `org.hibernate.boot.registry.BootstrapServiceRegistry`.
 The `BootstrapServiceRegistry` is intended to hold services that Hibernate needs at both bootstrap and run time.
 This boils down to 3 services:
 
-`org.hibernate.boot.registry.classloading.spi.ClassLoaderService`:: which controls how Hibernate interacts with `ClassLoader`s
+`org.hibernate.boot.registry.classloading.spi.ClassLoaderService`:: which controls how Hibernate interacts with ``ClassLoader``s.
```

Comment:
This is because of asciidoc gradle plugin's bug: https://stackoverflow.com/questions/34236730/how-to-use-a-closing-backtick-followed-by-an-alphabet-in-asciidoc

I solved the issue by replacing single backticks with double ones.

Reply:
nice, thanks :)